cli/cli_app.go

```diff
@@ -391,6 +391,22 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		terragruntOptions.RetryableErrors = terragruntConfig.RetryableErrors
 	}
 
+	if val := terragruntConfig.RetryMaxAttempts; val != nil {
```

Comment:
```suggestion
	if terragruntConfig.RetryMaxAttempts != nil {
```

Reply:
I do this assignment to reduce the verbosity in the conditional since I have to check if the value is less than 1 to return the error to the user. I could remove it, but the conditional would look a bit more messy. Are you sure?